index.html

```diff
@@ -452,18 +452,31 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 used to fetch the resource, as identified by the ALPN Protocol ID
 [[!RFC7301]]; resources retrieved from <a data-cite=
 "!HTML/#relevant-application-cache">relevant application caches</a>
-or local resources, return an empty string. The registry for
-protocol identifiers lists the possible values for the <a href=
+or local resources, return an empty string. When a proxy is
+configured, if a tunnel connection is established then this attribute
+MUST return the ALPN Protocol ID of the tunneled protocol, otherwise
+it MUST return the ALPN Protocol ID of the first hop to the proxy.
+In order to have precisely one way to represent any ALPN protocol ID,
+the following additional constraints apply: octets in the ALPN
+protocol MUST NOT be percent-encoded if they are valid token
+characters except "%", and when using percent-encoding, uppercase hex
+digits MUST be used.</p>
+<p>The registry for protocol identifiers lists the possible values
```

Comment:
The whole point of the issue is that the registry isn't the set of "possible" values, it's a set of values whose meaning is generally well-defined.  I'd say that the registry lists some common values, but implementations need to be prepared to encounter identifiers for experimental or unregistered protocols.